server/routes/api.py

```diff
@@ -244,6 +244,56 @@ def post_comment(author_id: int, post_id: int) -> Response:
     return Response(status=httpStatus.OK)
 
 
+@bp.route("/authors/<int:author_id>/followers", methods=["GET"])
+def get_followers(author_id: int) -> Response:
+    followers = Requests.query.filter_by(to=author_id).all()
+    return (
+        make_response(jsonify(type="followers", items=[f.json() for f in followers])),
+        httpStatus.OK,
+    )
+
+
+@bp.route("/authors/<int:author_id>/followers/<int:follower_id>", methods=["GET"])
+def is_follower(author_id: int, follower_id: int) -> Response:
```

Comment:
should we protect this endpoint with `login_required`?

Reply:
nope, we don't need to, the specs don't say nor imply we do